src/farcasterd/runtime.rs

```diff
@@ -134,8 +134,8 @@ pub struct Stats {
     punish: u64,
     abort: u64,
     initialized: u64,
-    awaiting_funding_btc: u64,
-    awaiting_funding_xmr: u64,
+    awaiting_funding_btc: HashSet<SwapId>,
+    awaiting_funding_xmr: HashSet<SwapId>,
```

Comment:
much less error prone & expressive as information source :)

I agree to keep the others 
https://github.com/farcaster-project/farcaster-node/blob/feeb691c737fe54815c00ef849c8b7b1fb05e664/src/farcasterd/runtime.rs#L139-L142
as `u64` to avoid state bloat for long running node.

Reply:
Also thought to switch to sets for all of them, but wanted to discuss it in another PR.

IMO it's not a problem even for long running nodes, and we could use another type (a queue for example) to keep the size constrained.